python/lsst/skymap/tractInfo.py

```diff
@@ -27,43 +30,52 @@
 
 from .patchInfo import PatchInfo, makeSkyPolygonFromBBox
 
-__all__ = ["TractInfo"]
-
 
 class TractInfo:
     """Information about a tract in a SkyMap sky pixelization
 
-    The tract is subdivided into rectangular patches. Each patch has the following properties:
-    - An inner region defined by an inner bounding. The inner regions of the patches exactly tile the tract,
-      and all inner regions have the same dimensions. The tract is made larger as required to make this work.
-    - An outer region defined by an outer bounding box. The outer region extends beyond the inner region
-      by patchBorder pixels in all directions, except there is no border at the edges of the tract.
-      Thus patches overlap each other but never extend off the tract. If you do not want any overlap
-      between adjacent patches then set patchBorder to 0.
+    Parameters
+    ----------
+    id : `int`
+        tract ID
+    patchInnerDimensions : `tuple` of `int`
+        Dimensions of inner region of patches (x,y pixels).
+    patchBorder : `int`
+        Overlap between adjacent patches (in pixels)
+    ctrCoord : `lsst.afw.geom.SpherePoint`
+        ICRS sky coordinate of center of inner region of tract; also used as
+        the CRVAL for the WCS.
+    vertexCoordList : `list` of `lsst.afw.geom.SpherePoint`
+        Of vertices that define the boundaries of the inner region.
+    tractOverlap : `lsst.geom.Angle`
+        Minimum overlap between adjacent sky tracts; this defines the minimum
+        distance the tract extends beyond the inner region in all directions.
+    wcs : `lsst.afw.image.Wcs`
+        The reference pixel will be shifted as required so that the lower
+        left-hand pixel (index 0,0) has pixel position 0.0, 0.0.
+
+    Notes
+    -----
+    The tract is subdivided into rectangular patches. Each patch has the
+    following properties:
+    - An inner region defined by an inner bounding. The inner regions of the
```

Comment:
"inner bounding" -> "inner bounding box"

Reply:
Also please make this render as an unordered list by inserting a blank line above this line.